test/end-to-end/Azure.Functions.NodejsWorker.E2E/Azure.Functions.NodejsWorker.E2E/CosmosDBEndToEndTests.cs

```diff
@@ -7,8 +7,16 @@
 
 namespace Azure.Functions.NodeJs.Tests.E2E
 {
+    [Collection(Constants.FunctionAppCollectionName)]
     public class CosmosDBEndToEndTests 
     {
+        private readonly FunctionAppFixture _fixture;
+
+        public CosmosDBEndToEndTests(FunctionAppFixture fixture)
+        {
+            this._fixture = fixture;
```

Comment:
Only use this fixture if FunctionsHostUrl  contains localhost. Otherwise, no need to spin up the host.